app/src/test/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -30,52 +29,31 @@ internal class PopularMoviesViewModelTest {
     )
     val getPopularMoviesUseCase: GetPopularMoviesUseCase = mockk()
     val popularMoviesFragmentNavigator: PopularMoviesFragmentNavigator = mockk()
-
-    @Test
-    fun `when getPopularMoviesUseCase called and load data should set popularMovies`() = runTest {
-        coEvery { getPopularMoviesUseCase() } returns moviesModel
-        val tested = PopularMoviesViewModel(getPopularMoviesUseCase, popularMoviesFragmentNavigator)
-
-        tested.popularMovies.value shouldBeEqualTo movies
+    val successState = UIState.Success(movies)
```

Comment:
i would inline usage of states, it is easier to find out in the tests in my opinion, especially when the all of the states are used only once, in one test, i would follow approach from this example:

`tested.popularMoviesViewState.value shouldBeEqualTo UIState.Loading`